benchmarks/run.py

```diff
@@ -60,7 +65,7 @@
 # in order of performance for csv
 other_tests = [
     t for t in
-    [TestCAttrs, TestValideer, TestMarshmallow, TestVoluptuous, TestTrafaret, TestDRF, TestCerberus]
+    [TestCAttrs, TestValideer, TestMarshmallow, TestVoluptuous, TestTrafaret, TestDRF, TestCerberus, TestColander]
```

Comment:
it's a bit hacky, but the benchmarks currently use this order to decide how the order results are shown in, therefore could you changed this to 
```suggestion
    [TestColander, TestCAttrs, TestValideer, TestMarshmallow, TestVoluptuous, TestTrafaret, TestDRF, TestCerberus]
```